couler/tests/__init__.py

```diff
@@ -10,3 +10,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import couler
+
+couler.config_defaults(name_salter=lambda x: x)
```

Comment:
This file should not be needed? The default identify function should be set already in the implementation instead.

Reply:
That's assuming we define the default salt as identity like we talked about in another comment here.

Reply:
What do we want the default salt function to be?
This code is here because if we use actual salting comparing YAMLs will become more complicated.